<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>521</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgb(0,0,0);
}

QMenuBar{
background-color: rgb(0,0,0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>751</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    background-color: rgb(0,0,0);
    border: 1px solid black;
    color: white; 
}</string>
    </property>
    <property name="title">
     <string>Suppliers</string>
    </property>
    <widget class="QTableWidget" name="supplierList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>401</width>
       <height>171</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>401</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
	background-color: rgb(0,0,0);
	gridline-color: rgb(255, 255, 255);
	color: rgb(255,255,255);
	border-radius: 10px;
	border: 2px solid rgb(255,255,255);
	selection-background-color: rgb(46, 46, 62);
}</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>SupplierID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Supplier Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>City</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Country</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Contact. No</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="supplierUpdateButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>400</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);
	
}

QPushButton:hover{
	border: 2px solid rgb(53, 53, 71)
}</string>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>731</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    background-color: rgb(0,0,0);
    border: 1px solid black;
    color: white; 
}</string>
     </property>
     <property name="title">
      <string>Search Supplier</string>
     </property>
     <widget class="QComboBox" name="supplierSearchCombo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox QAbstractItemView {
    background-color: rgb(30, 30, 40);
    color: #FFF;
    selection-background-color: rgb(53, 53, 71);
    selection-color: #FFF;
}


QComboBox{
	background-color: rgb(30, 30, 40);
	color: rgb(255,255,255);
	border: 2px solid rgb(37, 39, 48);
}

QComboBox:hover{
	border: 2px solid rgb(53, 53, 71);
	
	background-color: rgb(40, 40, 54);
}</string>
      </property>
      <item>
       <property name="text">
        <string>Name</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SupplierID</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="supplierSearchLine">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>421</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);

}

QLineEdit:hover{
	border: 2px solid rgb(53, 53, 71)
}

QLineEdit: focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supplierSearchButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);
	
}

QPushButton:hover{
	border: 2px solid rgb(53, 53, 71)
}</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="supplierDeleteButton">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>400</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);
	
}

QPushButton:hover{
	border: 2px solid rgb(53, 53, 71)
}c</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>731</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    background-color: rgb(0,0,0);
    border: 1px solid black;
    color: white; 
}</string>
     </property>
     <property name="title">
      <string>Add Supplier</string>
     </property>
     <widget class="QLineEdit" name="supplierAddName">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);

}

QLineEdit:hover{
	border: 2px solid rgb(53, 53, 71)
}

QLineEdit: focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="supplierAddContact">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);

}

QLineEdit:hover{
	border: 2px solid rgb(53, 53, 71)
}

QLineEdit: focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="supplierAddEmail">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);

}

QLineEdit:hover{
	border: 2px solid rgb(53, 53, 71)
}

QLineEdit: focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="supplierAddCity">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);

}

QLineEdit:hover{
	border: 2px solid rgb(53, 53, 71)
}

QLineEdit: focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: #FFF;
}</string>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: #FFF;
}</string>
      </property>
      <property name="text">
       <string>Contact Number:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: #FFF;
}</string>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: #FFF;
}</string>
      </property>
      <property name="text">
       <string>City:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supplierInsertButton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);
	
}

QPushButton:hover{
	border: 2px solid rgb(53, 53, 71)
}</string>
      </property>
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: #FFF;
}</string>
      </property>
      <property name="text">
       <string>Country:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="supplierAddCountry">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>20</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);

}

QLineEdit:hover{
	border: 2px solid rgb(53, 53, 71)
}

QLineEdit: focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);
	
}

QPushButton:hover{
	border: 2px solid rgb(53, 53, 71)
}</string>
     </property>
     <property name="text">
      <string>&lt;--</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
